### ISSUE TITLE
The ranges of BitcoinTransaction.locktime

### ISSUE THREAD

**Issue by ShookLyngs** [utxostack/rgbpp-explorer#20]
When the `BitcoinTransaction.locktime` field is not `0`, it means the output can only be spent if:
```
locktime < 500000000 && height > locktime ||
locktime >= 500000000 && time > locktime
```

And the [time](https://learnmeabitcoin.com/technical/block/time/) that Bitcoin uses is the number of seconds since 01 January 1970. It's a 10-digit integer, while our backend uses a 13-digit integer for timestamps by default. This means that when normalizing the `locktime` field, we use the `Date` type to define it in GraphQL, which can be inaccurate in two ways:
1. When `locktime < 500000000`, the number represents a block height instead of a timestamp
2. When `locktime >= 500000000`, the number represents a 10-digit timestamp instead of a 13-digit timestamp

This issue is reported by @Vibes-INS that it can happen in other APIs as well. I'm not sure if we should handle the value of `locktime` manually, or maybe we could change the type of the field to `Float` and leave it as it is, so at least the type of the field is not wrong.

If we want to align all timestamps' format returned by the backend, we could just add 3 zeros to the end of the target numbers, that is, if we can identify which fields need to be modified. The reason is when `locktime < 500000000`, the number shouldn't be modified as it represents a target block height, not a timestamp.

In conclusion, we have two questions:
1. Should the type of `BitcoinTransaction.locktime` be updated to `Float` instead of `Date`?
2. Should we align the format of timestamps? (e.g., ensure all timestamps are 13-digit integers)
